test/fixtures/v5/end_of_road/sharp_left_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Gire a cerrada a la izquierda",
         "fr": "Tourner franchement à gauche",
         "id": "Belok tajam kiri",
+        "it": "Gira a Leggera svolta a sinistra",
```

Comment:
If it isn’t possible to translate the component strings in a way that would make sense, it’s also possible to override this particular configuration using an [override file](https://github.com/Project-OSRM/osrm-text-instructions/tree/master/languages/overrides/).